S7.Net/Protocol/S7WriteMultiple.cs

```diff
@@ -0,0 +1,136 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using S7.Net.Types;
+
+namespace S7.Net.Protocol
+{
+    public static class S7WriteMultiple
+    {
+        public static int CreateRequest(byte[] pdu, DataItem[] dataItems)
```

Comment:
This seems to be used only internally. If that's the case, mark the class and method as internal.

Reply:
Yes it is. I'm refactoring this to improve readability/performance, will keep in mind to make it internal at some point.